src/metabase/util/polyfills.cljc

```diff
@@ -0,0 +1,6 @@
+(ns metabase.util.polyfills
+  #?@(:cljs-test ((:require ["@peculiar/webcrypto" :as webcrypto]))))
+
+;; The browser-compatible Crypto object and the `global.crypto` instance are unavailable in NodeJS before v23.0.0.
+;; Since both developers and Github CI may be using older Node versions, we install a Polyfill and include it here.
+#?(:cljs-test (do (println "Webcrypto polyfill!") (set! js/crypto (webcrypto/Crypto.))))
```

Comment:
nit: put an expiration date to goad someone into removing it in the future

```suggestion
#?(:cljs-test (do (println "Webcrypto polyfill! Remove me once NodeJS v23 is sufficiently ancient.") (set! js/crypto (webcrypto/Crypto.))))
```

Reply:
Good callout. I've added a detailed `TODO` comment in the code. I was going to make the `println` more prominent, but actually I only had that in there so I could confirm it was loading properly. So I've just removed the `println`; it's likely given the Node LTS for v22 (which still needs this polyfill) extends into 2027.